### ISSUE TITLE
Installed but doesnt work - How to fix?

### ISSUE THREAD

**Issue by ImpetuousRacer** [jlippold/tweakCompatible#15213]
I have 2 iPad Mini 2 both on 11.3.1 with Electra.

One works fine (used vfs exploit 1.0) on freshish restore

Other installs but there is no filter listed under all sources and no information on the bottom of each app (used multipath exploit 1.0.2) - this one also had older sources that loaded from prior jb.

Not sure how to get tweakcompatible to work on this device. It’s same hardware and iOS.

**Comment by jlippold** [comment]
Do other tweaks work on that bad device?

**Comment by ImpetuousRacer** [comment]
Yes they do. It appears everything else works smoothly.

I deleted all rogue non original repos.
I’ve rebooted and rejailbroke.
I’ve uninstalled and reinstalled your tweak as well. 

Any next steps I should look into? Thanks

Best,
James

**Comment by akaJayson** [comment]
Same issue on my iPhone SE with iOS 11.2.6, this works fine on my iPad 2017 model with 11.0.2

**Comment by jlippold** [comment]
Do you have little x by chance? I heard other people saying it conflicts

**Comment by ImpetuousRacer** [comment]
I do not. I mirrored the same packages and sources as the iPad it works on to see if it was a conflicting package as I saw someone mention an ad blocker app could cause an issue.

**Comment by jlippold** [comment]
Does tweakcompatible show up as the second entry in cydia on the sources tab?

**Comment by ImpetuousRacer** [comment]
It does not.

**Comment by jlippold** [comment]
man I'm stumped, do you have a syslog or anything?

Does the tweak called `Flame` work?

**Comment by ImpetuousRacer** [comment]
Ok, I got it figured out.

 

It looks like people who had Yalu jailbreak installed on iOS 10.2 and upgraded to 11.3.1, then jailbroke with Electra could have had this issue:

Source: https://www.reddit.com/r/jailbreak/comments/8wxt9f/questionsemirestore_11_lite_for_ios_1131_electra/

 

Then I followed this tutorial and how to unjailbreak the device with a tool that would fully remove all remnants (Unjailbreak tool from delectra):

https://www.youtube.com/watch?v=PyzpkZfZRpA <https://www.youtube.com/watch?v=PyzpkZfZRpA&feature=youtu.be> &feature=youtu.be

 

Then I rejailbroke with Electra.

 

Now I installed it and it is working. Therefore, I am not sure what the root of the problem was, but hopefully this can help anyone else in the future.

Thanks!

 

Best,

James
